bin/rake

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
```

Comment:
Whats this file about?

Reply:
This is an executable file which allows us to run the rake-cli that has commands like `rake ...`. Rake is a task runner that is used by rails.